test/fitnesse/junit/FitNesseRunnerExtensionTest.java

```diff
@@ -41,9 +43,12 @@ protected String getSuiteName(Class<?> klass) throws InitializationError {
 
     @Override
     protected FitNesseContext createContext(Class<?> suiteClass) throws Exception {
-      new PluginsClassLoader(getFitNesseRoot(suiteClass)).addPluginsToClassLoader();
+      ClassLoader classLoader = new PluginsClassLoaderFactory().getClassLoader(getFitNesseRoot(suiteClass));
+      ClassUtils.setClassLoader(classLoader);
```

Comment:
This line is not needed right? `makeFitNesseContext()` called below already ensures ClassUtils is properly set.